docs/bin/plugin_formatter.py

```diff
@@ -117,7 +117,7 @@ def write_data(text, options, outputname, module):
         fname = os.path.join(options.output_dir, outputname % module)
         fname = fname.replace(".py","")
         f = open(fname, 'w')
-        f.write(text.encode('utf-8'))
+        f.write(to_text(text))
         f.close()
```

Comment:
I'm looking into a larger scale change of opening with `'wb'` and ensuring that the data we are writing is bytes